rocky/rocky/views/mixins.py

```diff
@@ -488,6 +488,12 @@ def get_ooi_properties(self, ooi: OOI) -> dict:
             props = {"owner" if key == "user_id" else key: value for key, value in props.items()}
         else:
             props.pop("user_id")
+        if props["object_type"] == "Question":
+            config_pk = "Config|" + "|".join(str(self.ooi).split("|")[::-1][0:-1]).replace("/bit/", "")
```

Comment:
maybe this would be better placed as a helper property on the Question OOI object model?

Reply:
Adressed in 7c21ea30123a550a6f3917327693f3fe3ef60d4d